config/application.rb

```diff
@@ -28,6 +28,28 @@
 
 # Global config
 require_relative "../app/models/global_setting"
+
+# This custom inflector is needed because of our jobs directory structure.
+# Ideally, we should not prefix our jobs with a `Jobs` namespace but instead
+# have a `Job` suffix to follow the Rails conventions on naming.
+#
+# Based on:
+# https://github.com/rails/rails/blob/75e6c0ac/railties/lib/rails/autoloaders/inflector.rb#L7-L19
+# Additonally this allows plugins to override the inflections cause it is defined prior to plugin init
```

Comment:
If plugins are expected to use it, then IMO we should add a documented method to the plugin API in `Plugin::Instance`